README.md

```diff
@@ -11,19 +11,21 @@ To launch the tray app:
 ```
 $ netctl-tray
 ```
-Note: launching the app as root is not safe.
+Note: *launching the app as root is not safe*.
 
 ## Installation
 
-[This app is available on the AUR](https://aur.archlinux.org/packages/netctl-tray/)
+Note: if you use `netctl-auto`, add `--features auto` to the cargo build command
+
 ```
-$ git clone https://aur.archlinux.org/netctl-tray.git
-$ cd netctl-tray
-$ makepkg -si
+$ cargo build --release # --features auto if netctl-auto is used
+$ sudo ./install
 ```
 
 ## Troubleshooting
 
+If you have a `netctl` profile named `None` you may run into issues.
+
```

Comment:
how come? what causes it?

Reply:
This was true before. Your code set the active profile name to "none" and then iterated through the profiles to find the active one, then checked to see if the name was still "none". I changed it to "None" so that it would look nicer in notifications etc, and documented the issue. I can clean up the code with a variable instead of the original sentinel value design.